.copr/Makefile

```diff
@@ -42,6 +42,13 @@ srpm: deps
 	install -D -m644 $(PKGNAME)-$(VERSION)-$(RELEASE).tar.gz $(SOURCEDIR)
 	rpmbuild -bs $(spec)/rhc.spec
 	install -D -m644 $(SRCRPMDIR)/*.rpm $(outdir)/
+	install -D -m644 $(PKGNAME)-$(VERSION)-$(RELEASE).tar.gz $(outdir)/
 
+# This should not be part of Makefile. COPR should provide some
+# way how to install packages required for running
+# make -f .copr/Makefile srpm outdir=./out
 deps:
-	dnf install -y golang git rpm-build
+	rpm -qa | grep '^golang-[0-9]' \
+	&& rpm -qa | grep '^git-[0-9]' \
+	&& rpm -qa | grep '^rpm-build-[0-9]' \
+	|| dnf install -y golang git rpm-build
```

Comment:
while I can see what you want to achieve, IMHO this makes things more complicated:
- in addition to the package list in the `dnf install` command and in the `srpm_build_deps` list of the packit config, there are more package names in the `grep` commands
- running `rpm -qa` is slow...

Since all we need there are the spec file and the source tarball, I think the build can be simplified a lot:
- in `.copr/Makefile`:
   - move the spec file generation in its own target (eg `spec`), adding it as dependency of the `srpm` target; this would be the `sed` call
   - move the tarball generation in its own target (eg `tarball`), adding it as dependency of the `srpm` target; this would be only the `make dist` call, no need to also copy the tarball anywhere
- in the packit config:
   - run `make -f .copr/Makefile spec` as only step in `post-upstream-clone`
   - run `make -f .copr/Makefile tarball` as first step in `create-archive`
- drop `rpm-build` from `srpm_build_deps`, as it will not be needed anymore

Reply:
It was little bit more complicated, but I use proposed approach.